include/openssl/experimental/kem_deterministic_api.h

```diff
@@ -0,0 +1,59 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
```

Comment:
Please add this file to doc.go.

Reply:
How does [8b8caa6](https://github.com/aws/aws-lc/pull/1578/commits/8b8caa6475ba44c8d56788236b15812244962584) look?